lib/plugins/aws/provider.js

```diff
@@ -1816,7 +1817,21 @@ Object.defineProperties(
         const localTag = `${repositoryName}:${imageName}`;
         const remoteTag = `${repositoryUri}:${imageName}`;
 
-        const buildDockerArgs = ['build', '-t', localTag, '-f', pathToDockerfile, imagePath];
+        const buildArgsArr = [];
+        Object.keys(buildArgs).map((key) =>
```

Comment:
can we just assign the result of this map to a `buildArgsARr` variable instead of using `push` ?